packages/example/src/components/Notifications/History.tsx

```diff
@@ -0,0 +1,201 @@
+import type { Notification } from '@usedapp/core'
+import { ChainId, useNotificationsContext, useTransactionsContext } from '@usedapp/core'
+import type { ReactNode } from 'react'
+import styled from 'styled-components'
+import { TextBold } from '../../typography/Text'
+import { ContentBlock } from '../base/base'
+
+interface TableWrapperProps {
+  children: ReactNode
+  title: string
+}
+
+const TableWrapper = ({ children, title }: TableWrapperProps) => (
+  <ContentBlock>
+    <CellTitle>{title}</CellTitle>
+    <Table>{children}</Table>
+  </ContentBlock>
+)
+
+interface TransactionListProps {
+  chainId: ChainId
+}
+
+export const TransactionsList = ({ chainId }: TransactionListProps) => {
+  const { transactions } = useTransactionsContext()
+  const chainTransactions = transactions[chainId] ?? []
+  const reversed = [...chainTransactions].reverse()
+
+  return (
+    <TableWrapper title="Transactions history">
+      {reversed.map((tx) => (
+        <div key={tx.transaction.hash}>
+          {!tx.receipt && <Spinner />}
+          <span>{tx.transaction.hash.slice(0, 6) + '...' + tx.transaction.hash.slice(-4)}</span>
```

Comment:
Use [shortenAddress(...)](https://usedapp.readthedocs.io/en/latest/core.html#shortenaddress)

Reply:
ahhh, shall we add shortenTransaction equvalent?

Reply:
We can just remove checking if the address is correct (and checksumming it). Anyway, it should just work on any string which is long enough.